crates/test/src/rt/browser.rs

```diff
@@ -17,7 +17,7 @@ pub struct Browser {
 #[wasm_bindgen]
 extern "C" {
     type HTMLDocument;
-    static document: HTMLDocument;
+    static DOCUMENT: HTMLDocument;
```

Comment:
This still needs to be lowercase on the JS side:

```suggestion
    #[wasm_bindgen(js_name = document)]
    static DOCUMENT: HTMLDocument;
```